docs/Project-Configuration/Project-Commands.md

```diff
@@ -352,24 +352,26 @@ by a patch submission.
 
 Parameters:
 
--   `dir`: the directory to clone into
--   `revisions`: For commit builds, each module should be passed as
-    `<module_name> : ${<module_name>_rev}`. For patch builds, the hash
+- `dir`: the directory to clone into
+- `revisions`: For commit builds, each module should be passed as
```

Comment:
nit: do we want to keep the follow-on lines for a bullet point at the same indentation level? I slightly preferred when the markdown bullet stayed at the same indentation level. The other markdown lists in the file also keep the indentation the same.

Maybe one day we should have a markdown linter or something 🤔

Reply:
Ah yeah I'll have to adjust my editor to not do this